### PR TITLE
Add support for logarithmic charts.

### PR DESCRIPTION
**Pull request by keep94**
Allow the 'base' attribute of a chart to be set.

### CODE REVIEW

File: docs/resources/dashboard.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ The `chart` mapping supports the following:
 * `summarization` - (Required) Summarization strategy for the chart. MEAN is default. Valid options are, `MEAN`, 
 `MEDIAN`, `MIN`, `MAX`, `SUM`, `COUNT`, `LAST`, `FIRST`
 * `description` - (Optional) Description of the chart
+* `base` - (Optional) The base of logarithmic scale charts. Usually 10. Omit or set to 0 for linear scale.
```
Comment (markpeek) [inline docs/resources/dashboard.md:115]: Might want to clarify `Usually 10.`. That sounds like a default but then you go on to say `Omit or 0 for linear scale`. Perhaps saying `Omit or set to 0 for the default linear scale. Usually set to 10 for traditional logarithmic scale.`.

File: wavefront/resource_dashboard.go [hunk 2]
```diff
@@ -369,6 +369,11 @@ func resourceDashboard() *schema.Resource {
 					Required:    true,
 					Description: "Name of the Chart",
 				},
+				"base": {
+					Type:        schema.TypeInt,
+					Optional:    true,
+					Description: "Base of logarithmic scale, 0 for linear",
```
Comment (markpeek) [inline wavefront/resource_dashboard.go:375]: `Base of logarithmic scale, default is 0 for linear scale.`

**Review by markpeek — CHANGES_REQUESTED** [review]
A few wording changes requested to clarify.

File: docs/resources/dashboard.md (hunk 1 above)
Comment (keep94) [reply docs/resources/dashboard.md]: Thank you for the feedback. I changed the wording in the doc as you suggested.

File: wavefront/resource_dashboard.go (hunk 2 above)
Comment (keep94) [reply wavefront/resource_dashboard.go:375]: Reworded the description so that it is clear that 0 is also the default.

**Review by markpeek — APPROVED** [review]
LGTM

**Pull request merged**